common/djangoapps/microsite_configuration/backends/filebased.py

```diff
@@ -0,0 +1,199 @@
+"""
+Microsite backend that reads the configuration from a file
+
+"""
+import edxmako
+import os
+import threading
+
+from django.conf import settings
+from microsite_configuration.backends.base import AbstractBaseMicrositeBackend
+
+
+class SettingsFileMicrositeBackend(AbstractBaseMicrositeBackend):
+    """
+    Microsite backend that reads the microsites definitions
+    from a dictionary called MICROSITE_CONFIGURATION in the settings file.
+    """
+
+    def __init__(self, **kwargs):
+        super(SettingsFileMicrositeBackend, self).__init__(**kwargs)
+        self.current_request_configuration = threading.local()
+        self.current_request_configuration.data = {}
+        self.current_request_configuration.cache = {}
+
+    def has_configuration_set(self):
+        """
+        Returns whether there is any Microsite configuration settings
+        """
+        return getattr(settings, "MICROSITE_CONFIGURATION", False)
+
+    def get_configuration(self):
+        """
+        Returns the current request's microsite configuration
+        """
+        if not hasattr(self.current_request_configuration, 'data'):
+            return {}
```

Comment:
No i don't think so. This method return current request's microsite configuration if present as stated in docstring.